### PR TITLE
adding note to clarify execCommand() behavior on input and textarea

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1220696

Discussed with @ddbeck over slack.

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -4245,11 +4245,17 @@
             },
             "firefox": {
               "version_added": "1",
-              "notes": "From Firefox 82, nested calls are not supported (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>."
+              "notes": [
+                "From Firefox 82, nested calls are not supported (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>.",
+                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline api/Document.json:4250]: Just a couple nits (present tense and terseness)

```suggestion
                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

File: api/Document.json [hunk 2]
```diff
@@ -4245,11 +4245,17 @@
...
+              ]
             },
             "firefox_android": {
               "version_added": "4",
-              "notes": "From Firefox 82, nested calls are not supported (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>."
+              "notes": [
+                "From Firefox 82, nested calls are not supported (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>.",
+                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline api/Document.json:4257]:
```suggestion
                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

File: html/elements/input/input.json [hunk 3]
```diff
@@ -16,10 +16,12 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline html/elements/input/input.json:20]:
```suggestion
              "notes": "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

File: html/elements/input/input.json [hunk 4]
```diff
@@ -16,10 +16,12 @@
...
             },
             "firefox_android": {
-              "version_added": "4"
+              "version_added": "4",
+              "notes": "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline html/elements/input/input.json:24]:
```suggestion
              "notes": "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

File: html/elements/textarea.json [hunk 5]
```diff
@@ -19,14 +19,16 @@
               "version_added": true,
               "notes": [
                 "Before Firefox 6, when a <code>&lt;textarea&gt;</code> was focused, the insertion point was placed at the end of the text by default. Other major browsers place the insertion point at the beginning of the text.",
-                "A default background-image gradient is applied to all <code>&lt;textarea&gt;</code> elements, which can be disabled using <code>background-image: none</code>."
+                "A default background-image gradient is applied to all <code>&lt;textarea&gt;</code> elements, which can be disabled using <code>background-image: none</code>.",
+                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline html/elements/textarea.json:23]:
```suggestion
                "Before Firefox 89, manipulating the content of <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

File: html/elements/textarea.json [hunk 6]
```diff
@@ -19,14 +19,16 @@
...
               ]
             },
             "firefox_android": {
               "version_added": true,
               "notes": [
                 "Before Firefox 6, when a <code>&lt;textarea&gt;</code> was focused, the insertion point was placed at the end of the text by default. Other major browsers place the insertion point at the beginning of the text.",
-                "A default background-image gradient is applied to all <code>&lt;textarea&gt;</code> elements, which can be disabled using <code>background-image: none</code>."
+                "A default background-image gradient is applied to all <code>&lt;textarea&gt;</code> elements, which can be disabled using <code>background-image: none</code>.",
+                "Before Firefox 89, manipulating the content of <code>&lt;input&gt;</code> and <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands required workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a> for details)."
```
Comment (ddbeck) [inline html/elements/textarea.json:31]:
```suggestion
                "Before Firefox 89, manipulating the content of <code>&lt;textarea&gt;</code> elements using <code>Document.execCommand()</code> commands requires workarounds (see <a href='https://bugzil.la/1220696'>bug 1220696</a>)."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]

**Comment by chrisdavidmills** [comment]
Thanks @ddbeck I've committed all the suggestions, so we should be good to go now.

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**